SPECS/flatbuffers/flatbuffers.spec

```diff
@@ -0,0 +1,212 @@
+%global debug_package %{nil}
+# The .so version is explicitly constructed from project version—search
+# CMakeLists.txt for FlatBuffers_Library_SONAME_MAJOR and
+# FlatBuffers_Library_SONAME_FULL—but we manually repeat the SOVERSION here,
+# and use the macro in the file lists, as a reminder to avoid undetected .so
+# version bumps.
+%global so_version 2
+%global common_description %{expand:
+FlatBuffers is a cross platform serialization library architected for maximum
+memory efficiency. It allows you to directly access serialized data without
+parsing/unpacking it first, while still having great forwards/backwards
+compatibility.}
+Summary:        Memory efficient serialization library
+Name:           flatbuffers
+Version:        2.0.8
+Release:        1%{?dist}
+# The entire source code is Apache-2.0. Even code from grpc, which is
+# BSD-3-Clause in its upstream, is intended to be Apache-2.0 in this project.
+# (Google is the copyright holder for both projects, so it can relicense at
+# will.) See https://github.com/google/flatbuffers/pull/7073.
+License:        Apache-2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://google.github.io/flatbuffers
+Source0:        https://github.com/google/flatbuffers/archive/v%{version}/%{name}-%{version}.tar.gz
+BuildRequires:  cmake
+BuildRequires:  gcc-c++
+# The ninja backend should be slightly faster than make, with no disadvantages.
+BuildRequires:  ninja-build
+BuildRequires:  python3-devel
+BuildRequires:  python3-packaging
+BuildRequires:  python3-pip
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-wheel
+# From grpc/README.md:
+#
+#   NOTE: files in `src/` are shared with the GRPC project, and maintained
+#   there (any changes should be submitted to GRPC instead). These files are
+#   copied from GRPC, and work with both the Protobuf and FlatBuffers code
+#   generator.
+#
+# It’s not clearly documented which GPRC version is excerpted, but see
+# https://github.com/google/flatbuffers/pull/4305 for more details.
+#
+# It is not possible to unbundle this because private/internal APIs are used.
+Provides:       bundled(grpc)
```

Comment:
I don't think that's true, considering how we skimmed-down the spec in comparison to Fedora's original.